### DISCUSSION TITLE
Agenda for the Flatcar Office Hours on Tuesday, 9th of August 2022  17:30 CEST / 11:30am EDT / 8:30am PDT

### DISCUSSION THREAD

**Discussion by miao0miao** [flatcar/Flatcar#825] [Flatcar Office Hours]
## Agenda for the Flatcar Office Hours on Tuesday, 9th of August 2022  17:30 CEST / 11:30am EDT / 8:30am PDT

* Update: Now on Jitsi and not on Zoom: https://meet.element.io/JitsiJdgtwhchisudyeinvqvqsbwg
* Zoom link:  https://us06web.zoom.us/j/84336116610
  - Meeting ID: 843 3611 6610
  - Passcode: 444888
- Youtube live stream / recording: [link]()
- Community Calls calendar (all future calls): [link]()
  - iCal version: [link](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)

## Welcome
- Brief intro/check-in of the new participants in the Zoom call. Please introduce yourself and share what brings you here today.

## News
- future events: FrOSCon , https://www.froscon.org/en/
- past events:  SRE France @tormath1,  KCD Berlin recordings
- Reminder - AWS AMI deprecation 

## Release planning
- Ongoing and upcoming releases (https://github.com/orgs/flatcar-linux/projects/7/views/8)

## Q&A
- Questions from community participants, answered by the Flatcar maintainers

**Comment by tormath1** [comment]
I can also briefly talk about the event I'm organizing in France (https://sre-summercamp.github.io/) - if someone wants to submit a talk about Flatcar :partying_face:

**Comment by pothos** [comment]
We had to switch to Jitsi because of technical problems: https://meet.element.io/JitsiJdgtwhchisudyeinvqvqsbwg

**Reply by eumel8** [reply]
will there be a recording somewhere?

**Reply by tormath1** [reply]
Hi @eumel8, due to the technical problems mentioned we have not been able to record this session. Other office hours can be usually found on the Kinvolk Youtube channel: https://www.youtube.com/c/Kinvolk/videos
During this session we covered the items in this agenda, do you have any particular question regarding one of those?

**Reply by eumel8** [reply]
KCD Berlin videos was mentioned. Are there any news if and where they uploaded?

**Reply by tormath1** [reply]
Hi @eumel8, the videos are uploaded - you will find them there under: https://www.youtube.com/channel/UCZceOoRIWXglB24him9RpTg/videos. (Sorry for the delay :D)
